webapp/src/pages/Profile/Profile.tsx

```diff
@@ -25,10 +27,33 @@ const Profile = () => {
             );
         }
     }
+    const statusMap = new Map([
+        ['online', 'success'],
+        ['offline', 'error'],
+        ['in-game', 'primary']
+    ]);
+    const getColorStatus = (): any => {
+        const color = statusMap.get(user.current_status);
+        if (color === undefined) {
+            return 'error';
+        }
```

Comment:
@JulesMaisonneuve **info** c'est assez rependu d'utiliser `!` verifier si une valeur est `null` ou `undefined`

```suggestion
        if (!color) {
            return 'error';
        }
```

Reply:
Ok c'est changé